migrations/20190128134129-add-room-origin-column.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const TBL = 'Rooms';
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.addColumn(
+      TBL,
+      'origin', {
+        type: Sequelize.ENUM('auto_create', 'GUS', 'other', 'web'),
```

Comment:
@ConalCosgrove slack missing from the migration

Reply:
nice catch 👍🏻